src/core/internal/key.ts

```diff
@@ -834,6 +839,18 @@ export async function sign(
     }
     if (keyType === 'webauthn-p256') {
       const { credential, rpId } = key
+
+      const cacheKey = `webauthn_verified_${address}`
```

Comment:
We can narrow the cache key to the key's hash (id). There could be an instance where an account has 2 WebAuthn keys (ie. main key and a backup).

```suggestion
      const cacheKey = `webauthnVerified.${key.hash}`
```